package.json

```diff
@@ -65,10 +65,7 @@
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "*.{js,json,md}": [
-      "eslint --fix",
-      "prettier --write",
-      "git add"
-    ]
+    "*.{js}": ["eslint --fix", "git add"],
```

Comment:
Change this to `"*.js": ["eslint --fix", "prettier --write", "git add"],` and remove `js` from below?

Reply:
No problem. Done.